can_bus/json/BMS/BMS_tx.json

```diff
@@ -323,5 +323,18 @@
                 "unit": "degC"
             }
         }
+    },
+    "BMS_CommitInfo": {
```

Comment:
You can review the board name prefixes, they're added automatically now. 
This change just went in today so I still need to make a post about it

Reply:
Fixed!